single_kernel_mongo/managers/config.py

```diff
@@ -7,10 +7,15 @@
 import logging
 import time
 from abc import ABC, abstractmethod
+from functools import reduce
 from itertools import chain
+from pathlib import Path
+from typing import Any
 
+from deepmerge import always_merger
```

Comment:
TIL

Reply:
Me too tbh, I looked a while to find a good merging library :)